lldb/include/lldb/Target/Process.h

```diff
@@ -406,6 +406,36 @@ class Process : public std::enable_shared_from_this<Process>,
                                   lldb::StateType state);
   } Notifications;
 
+  class ProcessMemoryIterator {
```

Comment:
AFAICT, this isn't necessary anymore. If you think it's useful, you can make it a separate patch.